index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <header>My logo</header>
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+    <main class="main-section">
+      <h1>
+        I'm Juan<br />
+        Glad to see you!
+      </h1>
+      <h3>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </h3>
+      <section class="second-section">
```

Comment:
- [ ] :+1: Do you agree that this part is not a standalone section? If yes, as I think it should be, I am suggesting to you that the entire part of this code after the `header` section should a whole section, and I am proposing that we move this opening tag to line 20 so that you have the `h1` , `p`, etc. nested within the section. How about that? :wink:

Reply:
Hi @AbayomiOlaoye,

Could you please provide a more deep example?. You're suggesting this "I am suggesting to you that the entire part of this code after the header section should a whole section", however I nested everything in this section `<main class="main-section">`. Thanks in advance.

Reply:
Simply put, an important part of your page like page heading and paragraphs should not be in the same parent element with your navigation lists, and in this case, it's a header tag.

So, I'm saying your h1 and p tags should not be in the header section.

Reply:
Let me know if that seems cryptic to you again, 🙏

Reply:
Got it!